ci-operator/step-registry/azure/deprovision/resourcegroup/azure-deprovision-resourcegroup-commands.sh

```diff
@@ -59,3 +59,23 @@ for rg_file in "${rg_files[@]}"; do
         fi
     fi
 done
+
+# Remove resource group across subscriptions, and "az account set --subscription <cross subscription>" is used
+# Please better to ensure this is kept at the end of the script.
+if [ -f ${SHARED_DIR}/resourcegroup_cross-sub ]; then
+    # Exit with failure to ensure no leftovers are left unnoticed.  
+    if [[ ! -f "${CLUSTER_PROFILE_DIR}/azure-sp-contributor.json" ]]; then
+        echo "Error: Expected service principal not found. Cannot remove resources across subscriptions."
+        exit 1
+    fi
+    echo "Setting AZURE credential with Contributor role only for installer"
```

Comment:
The message is confused here, might something like "Removing the resource group from subscription <subscritpion_id>...."?

Reply:
Oh, bad copy paste, updated.